datamade/la-metro-councilmatic
# Adjusting autocomplete for IE 11

## Overview

This PR fixes the formatting of a JS function so that it is executable by IE.

### Checklist



## Testing Instructions

Run the website as usual on your local machine.
Search for 'Metro Green Line' and confirm the autocomplete works.

Handles #473
